launcher/updatedialog_moc.cpp

```diff
@@ -53,14 +52,12 @@ UpdateDialog::UpdateDialog(QWidget *parent, bool calledManually) :
 #endif
 	
 	QString url = QString::fromStdString(settings["launcher"]["updateConfigUrl"].String());
-	
-	QScopedPointer<QNetworkAccessManager> manager(new QNetworkAccessManager);
-	
-	QNetworkReply *response = manager->get(QNetworkRequest(QUrl(url)));
+		
+	QNetworkReply *response = networkManager.get(QNetworkRequest(QUrl(url)));
 	
 	QObject::connect(response, &QNetworkReply::finished, [&, response]{
```

Comment:
```suggestion
	connect(response, &QNetworkReply::finished, [&, response]{
```

FYI using `QObject::connect` is needed only when you're not inside a QObject-derived class (like `main` function)